tests/functional/testAuthenticationInBrowser.js

```diff
@@ -33,7 +33,7 @@ describe('User authentication in browser', function() {
     await vpn.waitForQuery(queries.screenInitialize.SIGN_UP_BUTTON.visible());
   });
 
-  it('Completes authentication', async () => {
+  it.only('Completes authentication', async () => {
```

Comment:
Revert this

Reply:
ah great catch, was doing to debugging previously, and this slipped through